### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by aalexren**
The new functionality has been added. Check it out, please.

### CODE REVIEW

File: HW2/events/FireAlarm.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#ifndef HW2_FIREALARM_H
+#define HW2_FIREALARM_H
+
+#include <string>
+#include <vector>
+
+//#include "Observer.h"
+
+// Moreover, I use Observer pattern to notify rooms when fire is hold.
+// Rooms – Observers, FireAlarm – Subject
```
Comment (l1va) [inline HW2/events/FireAlarm.h:14]: like

File: HW2/users/Student.h [hunk 2]
```diff
@@ -8,8 +8,9 @@
 #include <string>
 
 #include "User.h"
+#include "IGetWages.h"
 
-class Student : public User {
+class Student : public User, public IGetWages {
```
Comment (l1va) [inline HW2/users/Student.h:13]: lucky you

**Review by l1va — APPROVED** [review]

File: HW2/users/Student.h (hunk 2 above)
Comment (aalexren) [reply HW2/users/Student.h:13]: What do you mean?
@l1va

Comment (l1va) [reply HW2/users/Student.h:13]: get wages =)

**Pull request merged**